project/Dependencies.scala

```diff
@@ -17,10 +19,11 @@ object Dependencies {
   val awsSimpleEmail = "com.amazonaws" % "aws-java-sdk-ses" % "1.9.24"
   val snowPlow = "com.snowplowanalytics" % "snowplow-java-tracker" % "0.5.2-SNAPSHOT"
   val bCrypt = "com.github.t3hnar" %% "scala-bcrypt" % "2.4"
+  val s3 =  "com.amazonaws" % "aws-java-sdk-s3" % "1.9.30"
```

Comment:
Can you extract the awsVersion out into a variable, made common with awsSimpleEmail version?